library/src/com/google/maps/android/geojson/GeoJsonLineStringStyle.java

```diff
@@ -0,0 +1,168 @@
+    }
+
+    /**
+     * Gets the color of the GeoJsonLineString
+     *
+     * @return color of the GeoJsonLineString
+     */
+    public int getColor() {
+        return mPolylineOptions.getColor();
+    }
+
+    /**
+     * Sets the color of the GeoJsonLineString
+     *
+     * @param color color value of the GeoJsonLineString
+     */
+    public void setColor(int color) {
+        mPolylineOptions.color(color);
+        styleChanged();
+    }
+
+    /**
+     * Gets whether the GeoJsonLineString is geodesic
+     *
+     * @return true if GeoJsonLineString is geodesic, false otherwise
+     */
+    public boolean isGeodesic() {
+        return mPolylineOptions.isGeodesic();
+    }
+
+    /**
+     * Sets whether the GeoJsonLineString is geodesic
+     *
+     * @param geodesic true if GeoJsonLineString is geodesic, false otherwise
+     */
+    public void setGeodesic(boolean geodesic) {
+        mPolylineOptions.geodesic(geodesic);
+        styleChanged();
+    }
+
+    /**
+     * Gets the width of the GeoJsonLineString
+     *
+     * @return width of the GeoJsonLineString
+     */
+    public float getWidth() {
+        return mPolylineOptions.getWidth();
+    }
+
+    /**
+     * Sets the width of the GeoJsonLineString
+     *
+     * @param width width value of the GeoJsonLineString
+     */
+    public void setWidth(float width) {
+        mPolylineOptions.width(width);
+        styleChanged();
+    }
+
+    /**
+     * Gets the z index of the GeoJsonLineString
+     *
+     * @return z index of the GeoJsonLineString
+     */
+    public float getZIndex() {
+        return mPolylineOptions.getZIndex();
+    }
+
+    /**
+     * Sets the z index of the GeoJsonLineString
+     *
+     * @param zIndex z index value of the GeoJsonLineString
+     */
+    public void setZIndex(float zIndex) {
+        mPolylineOptions.zIndex(zIndex);
+        styleChanged();
+    }
+
+    /**
+     * Gets whether the GeoJsonLineString is visible
+     *
+     * @return true if the GeoJsonLineString visible, false if not visible
+     */
+    @Override
+    public boolean isVisible() {
+        return mPolylineOptions.isVisible();
+    }
+
+    /**
+     * Sets whether the GeoJsonLineString is visible
+     *
+     * @param visible true if the GeoJsonLineString is visible, false if not visible
+     */
+    @Override
+    public void setVisible(boolean visible) {
+        mPolylineOptions.visible(visible);
+        styleChanged();
+    }
+
+    /**
+     * Notifies the observers, GeoJsonFeature objects, that the style has changed. Indicates to the
+     * GeoJsonFeature that it should check whether a redraw is needed for the feature.
+     */
+    private void styleChanged() {
+        setChanged();
+        notifyObservers();
+    }
+
+    /**
+     * Gets a new PolylineOptions object containing styles for the GeoJsonLineString
+     *
+     * @return new PolylineOptions object
+     */
+    public PolylineOptions getPolylineOptions() {
```

Comment:
Convention for something like this is 'toPolylineOptions()', as 'get..' implies a local property of this object. Whereas what's happening here is a conversion.